### PR TITLE
Capstone Project

### PR DESCRIPTION
**Pull request by GeekyHacks**
In this Pull-Requst I did the following:
- Add mobile and desktop versions html&css layout 
- Made menu button, view more button interactive with Js
- Generate figures dynamically
- Fix linter Errors

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,17 @@
-# Microverse_HTML_CSS_JS-Template
+# Yemen National Youth Party
+
```
Comment (1GORDON) [inline README.md:1]: Great job @GeekyHacks 🤝

- [ ] As specified in the project requirements, Kindly ensure your README contains an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

File: README.md [hunk 2]
```diff
@@ -1,12 +1,17 @@
...
 
 <a name="readme-top"></a>
```
Comment (1GORDON) [inline README.md]: Great job @GeekyHacks 🤝

- [ ] As specified in the project requirements, Kindly deploy your project with the use of either GitHub pages or Netilify and add the deployment link to your readme file.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+        <div class="menuPageContent" id="menuPageContent">
+          <img class="menuX" src="./images/x.svg" alt="close" />
+          <ul class="menuPageList">
+            <li>
+              <a href="about.html">About</a>
+            </li>
+            <li>
+              <a href="#Scd_page">Program</a>
+            </li>
+            <li>
```
Comment (1GORDON) [inline index.html:43]: Great design @GeekyHacks 🤝

- [ ] Kindly ensure your design is a close replica of the design given. At the moment your design is completely different from the design provided. Kindly take a look at the [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) and refactor your project to match it.

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @GeekyHacks ✋

Good job so far!

## Highlights!
- Followed GitHub Flow ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @GeekyHacks,

✔️ Deployed app and nice design

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**